travel-event-host-app/src/app/searchevent/[keyword]/page.tsx

```diff
@@ -0,0 +1,4 @@
+import SearchSection from '@/components/searchSection/searchSection';
```

Comment:
for api route, why not put the search under the event route?
`api/event/search/[keyword]`